### DISCUSSION TITLE
Is it possible to created Nested Menus

### DISCUSSION THREAD

**Discussion by ghost** [thuliteio/doks#464] [Q&A]
I Think the Theme is amazing. I only can't get nested menu items to work under `docs`. 

I would like to have multiple collapse menus. Top menus are defined in config. and sublevels are decided with the 

```
menu:
  docs:
    parent: "child-1"
```

So it would create something like this in the sidebar:

```
- top-level
  - child-1
    - grand-child-1
      - unlimited levels (with option to hide, using the hidden = true option)
```

**Comment by h-enk** [comment]
Yes it's possible, but I can't help you now (my prio is on shipping versioning).

Extended menus are set for the [v0.4.0 Milestone](https://github.com/h-enk/doks/milestone/3) — see also: https://github.com/h-enk/doks/discussions/464 + https://github.com/h-enk/doks/issues/159

**Reply by ghost** [reply]
The Navbar solution is not something i'm searching for. But thanks for the answer. 

Any idea when you will be working on it. and 0.4.0 will be released?

**Reply by dhe365** [reply]
Is there an update on nested menus? Would be a really nice feature.

**Reply by h-enk** [reply]
Hmm, not really, but a new release will be available soon (not sure yet if I have the time to include this feature though — I'm a bit busy with other stuff).
